docs/Learning-Environment-Design-Agents.md

```diff
@@ -88,81 +100,135 @@ when certain game or simulation events occur, such as in a turn-based game,
 should call `Agent.RequestDecision()` manually.
 
 ## Observations and Sensors
+In order for an agent to learn, the observations should include all the
```

Comment:
Most of the changes in this section are moved blocks of the old version. I'll call out brand new sections.

In general, this part of the docs was written before sensors, so the only real distinction between observations was vector vs visual. Now there are multiple ways to generate vector observations, so I tried to pull out the generic parts from the CollectObservations documentation. Definitely open to suggestions on how to arrange this better.